quic/s2n-quic-core/src/recovery/hybrid_slow_start.rs

```diff
@@ -17,6 +17,11 @@ pub struct HybridSlowStart {
     pub(super) threshold: f32,
     max_datagram_size: u16,
     rtt_round_end_time: Option<Timestamp>,
+    use_hystart: bool,
+    pub(super) ss_growth_divisor: f32,
```

Comment:
yeah, so I'm this new function (could be something like `fn new_cwnd(&self, cwnd: f32, sent_bytes: usize) -> f32`  in `hybrid_slow_start.rs` would be called from `cubic.on_packet_ack`, instead of having the `self.congestion_window += (sent_bytes as f32) / self.slow_start.ss_growth_divisor;` logic be in `cubic.rs`

Reply:
OK. let me think about this.